src/main/java/module-info.java

```diff
@@ -65,4 +65,5 @@
     requires de.saxsys.mvvmfx.validation;
     requires richtextfx;
     requires unirest.java;
+    requires json;
```

Comment:
Just on comment here: please do not use org.json as we are getting license troubles when using this library with more strict Linux distributions. See https://github.com/JabRef/jabref/issues/3703 for some details.
An alternative should be `org.gson`

Reply:
I will remove this line because it really is unnecessary. 
But i would like to stick with org.json in this PR and come up with a follow up PR, because org.json is already used in other fetchers. I would like to replace them all together. I propose javax.json as library because the API is nearly the same. GPL 2.0 should give no problems.

I will fix #3703, if @Siedlerchr hasn't dived in already.

Reply:
I would go with GSON then in a next step because the unirest people use gson internally as well.
https://itsallbinary.com/jackson-vs-gson-vs-json-b-vs-json-p-vs-org-json-vs-jsonpath-java-json-libraries-features-comparison/
Maybe you can directly use the unirest-json implementation?
In any case we should write an ADR

Reply:
I have opened a follow up pr #5511 that can be merged after this one is merged.